src/js/models/tools/ellipse.js

```diff
@@ -47,7 +47,7 @@ define(function (require, exports, module) {
             backspaceKeyPolicy = new KeyboardEventPolicy(UI.policyAction.PROPAGATE_TO_BROWSER,
                 OS.eventKind.KEY_DOWN, null, OS.eventKeyCode.BACKSPACE);
         
-        Tool.call(this, "ellipse", "Ellipse", "ellipseTool", "tool.ellipse.select");
+        Tool.call(this, "ellipse", "Ellipse Tool", "ellipseTool", "tool.ellipse.select");
```

Comment:
I thought that the tool names in these constructors were vestigial and unused? @DivyaPrabhakar: do you know if these are the strings that show up in the search UI? If so, that's a problem because they're not localized.